lib/tasks/server/middleware/tests-server/index.js

```diff
@@ -14,10 +16,9 @@ TestsServerAddon.prototype.serverMiddleware = function(config) {
 
   var baseURL = cleanBaseURL(options.baseURL);
```

Comment:
baseUrl is either `/test` or `/`, so I got rid of checking whether it matched `test`. This code path was never tested anyway (I verified by throwing inside the `else`.